packages/quantic/playwright.config.ts

```diff
@@ -0,0 +1,53 @@
+import {defineConfig, devices} from '@playwright/test';
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+import dotenv from 'dotenv';
+import path from 'path';
+dotenv.config({path: path.resolve(__dirname, '.env')});
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  testDir: './force-app',
+  testMatch: '*.e2e.ts',
+  /* Run tests in files in parallel */
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: process.env.CI ? 2 : 0,
+  /* Opt out of parallel tests on CI. */
+  workers: process.env.CI ? 1 : undefined,
```

Comment:
That's just the default value that was set when creating the project, usually it's set to 1 because resources are limited in the CI and we wanna avoid flakiness, but it's a decision we can take discuss together.